charts/composer/templates/configmap.yaml

```diff
@@ -30,6 +29,8 @@ data:
   OTEL_EXPORTER_OTLP_TRACE_HEADERS: "{{ tpl .Values.otel.traceHeaders . }}"
   OTEL_SERVICE_NAME: "{{ tpl .Values.otel.serviceName . }}"
   {{- if not .Values.global.dev }}
+  ASTRIA_COMPOSER_SEQUENCER_ABCI_ENDPOINT: "{{ tpl .Values.config.sequencerRpc . }}"
```

Comment:
when `.Values.global.dev` sets to true it marks a breaking change to configs. Therefore, new env variables should be added to `else` bracket while values being removed should be added to `{{- if not .Values.global.dev }} ` bracket.